app/assets/javascripts/discourse/controllers/full-page-search.js.es6

```diff
@@ -9,8 +9,9 @@ import { iconHTML } from 'discourse-common/helpers/fa-icon';
 const SortOrders = [
   {name: I18n.t('search.relevance'), id: 0},
   {name: I18n.t('search.latest_post'), id: 1, term: 'order:latest'},
-  {name: I18n.t('search.most_liked'), id: 2, term: 'order:likes'},
-  {name: I18n.t('search.most_viewed'), id: 3, term: 'order:views'},
+  {name: I18n.t('search.latest_topic'), id: 2, term: 'order:latest_topic'},
```

Comment:
You shouldn't change the `id`s here for previous values for backwards compatibility.